### PR TITLE
Hopefully fixes another source of long timer singlecaps

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Very similar to 87fbe4c (#62415)

Got a ping from mothblocks complaining about "Mass bombing" and "Unplayable rounds" on sybil, caused by long timer singlecaps.
I'm like 90% sure they're using bz's slow heating to achieve this. So I'm just gonna cap bz production at 20 degrees above room temp. It's not like you're going to be doing fabrication at this temperature anyway, so this isn't horribly impactful.

For the curious among you, bz scales its temperature increase inversely to its pressure, among other factors.
This scaling is VERY harsh, so at high pressures it barely creates any heat at all. This is useful for generating horribly long delays

Adds gasmix logging to tank explosions
Hopefully this will be useful if I ever keel over and die someday

## Why It's Good For The Game

Insert discussion of single caps, cursed problems, and salt prs here. 
You know if you guys weren't absolute chodes about this stuff I wouldn't even notice

## Changelog

:cl:
del: Removed another source of long timer singlecaps
/:cl:

### CODE REVIEW

File: code/game/objects/items/tanks/tanks.dm [hunk 1]
```diff
@@ -318,6 +318,7 @@
 		air_contents.react(src)
 		pressure = air_contents.return_pressure()
 
+		log_atmos("[src.type] exploded with a mix of ", air_contents)
```
Comment (TemporalOroboros) [inline code/game/objects/items/tanks/tanks.dm:321]: You might want to also log the power of the explosion for convenience.

**Comment by TemporalOroboros** [comment]
Hold on, I need to adjust my timer mix so it's fully depleted by the time it goes off.

**Comment by LemonInTheDark** [comment]
Atmos coders man

File: code/game/objects/items/tanks/tanks.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/game/objects/items/tanks/tanks.dm:321]: Good idea

**Comment by Shadowflame909** [comment]
mfw i roll antag only to get blown up by an atmos bomber antag

Cuban Pete is back..and they're in atmos 😭

File: code/game/objects/items/tanks/tanks.dm [hunk 2]
```diff
@@ -320,6 +320,7 @@
 
 		// As of writing this this is calibrated to maxcap at 140L and 160atm.
 		var/power = (air_contents.volume * (pressure - TANK_FRAGMENT_PRESSURE)) / TANK_FRAGMENT_SCALE
+		log_atmos("[src.type] exploded with a power of [power] and a mix of ", air_contents)
```
Comment (Mothblocks) [inline code/game/objects/items/tanks/tanks.dm:323]:
```suggestion
		log_atmos("[type] exploded with a power of [power] and a mix of ", air_contents)
```

**Pull request merged**